src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -352,6 +352,336 @@
   </div>
 </div>
 
+{% if responseType equals "FULL_ADMISSION" %}
+{% if paymentOption is defined %}
+<div class="section">
+  <div class="section-heading">
+    <strong>When the defendant will pay</strong>
+  </div>
+    <div class="section-body">
+      <table>
+        <tr>
+          <td><strong>When will they pay:</strong></td>
+          <td>
+            {{ paymentOption }}
+          </td>
+        </tr>
+
+        {% if repaymentPlan is defined and repaymentPlan.repaymentOption == "By instalments" %}
+        <tr class="no-border">
+          <td><strong>Regular payments of:</strong></td>
+          <td>
+            {{ repaymentPlan.instalmentAmount }}
+          </td>
+        </tr>
+        <tr class="no-border">
+          <td><strong>Frequency of payments:</strong></td>
+          <td>
+            {{ repaymentPlan.paymentSchedule }}
+          </td>
+        </tr>
+        <tr class="no-border">
+          <td><strong>Make the first payment on:</strong></td>
+          <td>
+            {{ repaymentPlan.firstPaymentDate }}
+          </td>
+        </tr>
+        {% endif %}
+        {% if statementOfMeans is defined %}
+        <tr class="no-border">
+          <td><strong>Explain why you can’t pay the full amount now:</strong></td>
+          <td>
+            {{ statementOfMeans.reason}}
+          </td>
+        </tr>
+        {% endif %}
+      </table>
+    </div>
+</div>
+{% endif %}
+
+{% if paymentOption == 'By instalments' or  paymentOption == 'By a set date' %}
+<div class="section">
+  <div class="section-heading">
+    <strong>The defendant’s financial details</strong>
+  </div>
+  <div class="section-body">
+    <table>
+      <tr class="no-border">
+        <td><strong>Where are they living?</strong></td>
+        <td>
+          {{ residenceTypeDescription }}
```

Comment:
No need to say what type of housing it is if user selected OTHER? Can you double check with BA/UX/UC?

Reply:
done